### PR TITLE
drakrun: Refactor tool usage

### PR DESCRIPTION
**Pull request by chivay**
* Move vmi-win-offsets and get-explorer-pid calls into separate functions.
* Create dataclasses for handling runtime information

### CODE REVIEW

File: drakrun/drakrun/util.py [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+        return VmiOffsets(**vals)
+
+
+@dataclass_json
+@dataclass
+class RuntimeInfo:
+    vmi_offsets: VmiOffsets
+    inject_pid: int
+
+    def load(file_obj) -> 'RuntimeInfo':
```
Comment (BonusPlay) [inline drakrun/drakrun/util.py:54]: any reason to pass file_obj here instead of raw string?

**Review by BonusPlay — APPROVED** [review]
Just code review, untested. I like the `dataclasses_json` usage, should learn some myself.

File: drakrun/drakrun/util.py (hunk 1 above)
Comment (chivay) [reply drakrun/drakrun/util.py:54]: I wanted to follow the API of `json` module :thinking:

Comment (BonusPlay) [reply drakrun/drakrun/util.py:54]: `IO[AnyStr]` type hint then?

Comment (chivay) [reply drakrun/drakrun/util.py:54]: :trudne:

**Review by icedevml — APPROVED** [review]

File: drakrun/drakrun/util.py (hunk 1 above)
Comment (chivay) [reply drakrun/drakrun/util.py]: fix'd

**Pull request merged**